finish_release.sh

```diff
@@ -34,7 +34,7 @@ merge_release () {
     echo "Merge release to master"
     git checkout -q master
     git pull
-    git merge release
+    git merge release -m "Merge branch 'release'"
```

Comment:
I think you can do `--no-edit` as well, which will use whatever Git autogenerates as a message